### PR TITLE
[PHP] Updated PHP Syntax to include PHP 7+ functions

### PR DESCRIPTION
**Pull request by CameronHall**
Added functions mentioned in PHP 7 Changelog
- oci_set_call_timeout
- oci_set_db_operation
- gmp_binomial
- gmp_lcm
- gmp_perfect_power
- gmp_kronecker
- openssl_pkey_derive
- is_countable
- hrtime
- ftp_append
- spl_object_id
- hash_hkdf
- is_iterable
- socket_export_stream
- error_clear_last
- http_response_code

### CODE REVIEW

File: PHP/PHP Source.sublime-syntax [hunk 1]
```diff
@@ -1349,7 +1349,7 @@ contexts:
...
       scope: support.function.curl.php
-    - match: (?i)\b(str(totime|ptime|ftime)|checkdate|time(zone_(name_(from_abbr|get)|transitions_get|identifiers_list|o(pen|ffset_get)|version_get|location_get|abbreviations_list))?|idate|date(_(su(n(set|_info|rise)|b)|create(_from_format)?|time(stamp_(set|get)|zone_(set|get)|_set)|i(sodate_set|nterval_(create_from_date_string|format))|offset_get|d(iff|efault_timezone_(set|get)|ate_set)|parse(_from_format)?|format|add|get_last_errors|modify))?|localtime|g(et(timeofday|date)|m(strftime|date|mktime))|m(icrotime|ktime))\b
+    - match: (?i)\b(str(totime|ptime|ftime)|checkdate|(hr)?time(zone_(name_(from_abbr|get)|transitions_get|identifiers_list|o(pen|ffset_get)|version_get|location_get|abbreviations_list))?|idate|date(_(su(n(set|_info|rise)|b)|create(_from_format)?|time(stamp_(set|get)|zone_(set|get)|_set)|i(sodate_set|nterval_(create_from_date_string|format))|offset_get|d(iff|efault_timezone_(set|get)|ate_set)|parse(_from_format)?|format|add|get_last_errors|modify))?|localtime|g(et(timeofday|date)|m(strftime|date|mktime))|m(icrotime|ktime))\b
```
Comment (wbond) [inline PHP/PHP Source.sublime-syntax]: These condensed regexes are unfortunately rather difficult to read and write. We should really expand, them, but that is not necessary for this PR.

This change results in all of the various `time*()` functions having an `hrtime*()` variant. For example `hrtimezone_ abbreviations_list()` is marked as a support function, whereas it doesn't actually exist.

Comment (CameronHall) [reply PHP/PHP Source.sublime-syntax]: I've updated this now :)

@jfcherng I'll run your script and submit a new PR once this one is approved.

**Comment by CameronHall** [comment]
Good catch! Let me know if you spot any more issues @wbond 👍

**Comment by wbond** [comment]
Looks good, thanks @CameronHall!

**Pull request merged**